models/project/project.go

```diff
@@ -144,10 +144,21 @@ func (p *Project) Link() string {
 	return ""
 }
 
+func (p *Project) Icon() string {
```

Comment:
Could we make the name `IconName` or even `SvgIconName`?

* `{{svg name=...}}` : this function only provides the name of the SVG icon, it doesn't return the Icon image itself.
* Longer names make it easier to be search across the whole codebase, it's friendly for reviewing and refactoring.

(just a question, not blocker)

Reply:
I think `IconName` is enough, as the image type may not be SVG?
In my mind, `{{svg name=...}}` means `{{imgtype imgname=...}}`

ps: I have renamed it to `IconName`